routes/paypal.js

```diff
@@ -135,23 +142,14 @@ function doExpressCheckoutPayment(checkoutDetails, recurring, callback) {
 }
 
 var paypalRoutes = {
-  setupSingle: function(transaction, callback) {
```

Comment:
These all got simplified. Since we pass frequency in options now, I saw no more need in having these frequency based functions.